app/lib/research_data_harvester.rb

```diff
@@ -36,11 +37,21 @@ def harvest(collection)
   # Convenience method to harvest and index all collections in the config file
   # @example
   #   ResearchDataHarvester.harvest
-  def self.harvest
+  def self.harvest(use_cache = false)
+    Rails.logger.info "Harvesting and indexing research data collections"
+
+    unless use_cache
+      # Fetch latest community information from DataSpace
+      communities = DataspaceCommunities.new
+      File.write(CACHE_COMMUNITIES_FILE, JSON.pretty_generate(communities.tree))
+    end
```

Comment:
We only fetch a new list of communities from DataSpace when we reindex the data. See comment below.